modules/message-lane/src/inbound_lane.rs

```diff
@@ -97,8 +99,14 @@ impl<S: InboundLaneStorage> InboundLane<S> {
 			return false;
 		}
 
+		// if there are more unrewarded relayer entries than we may accept, reject this message
+		if self.storage.max_unrewarded_relayer_entries() <= data.relayers.len() as MessageNonce {
```

Comment:
```suggestion
		if data.relayers.len() as MessageNonce >= self.storage.max_unrewarded_relayer_entries() {
```
easier to parse if it matches the docs. Why do we disallow equality here?

Reply:
Because if there are already max entries, we can't add any more entries if relayer is different. And if relayer is the same, then we're rejecting message because of your suggestion [here](https://github.com/paritytech/parity-bridges-common/pull/418#pullrequestreview-511667612).